_browser/hello-blockstack.md

```diff
@@ -259,10 +259,10 @@ The application handles these situations as followed:
 
 ```js
 if (blockstack.isUserSignedIn()) {
-  var profile = blockstack.loadUserData().profile
+  var profile = blockstack.UserSession.loadUserData().profile
```

Comment:
`UserSession` can't be used like this. `loadUserData()` is an instance method. Should first create a `userSession` object.